server/src/domain/server-info/server-info.dto.ts

```diff
@@ -80,6 +80,7 @@ export class ServerMediaTypesResponseDto {
 }
 
 export class ServerFeaturesDto implements FeatureFlags {
+  isConfigFile!: boolean;
```

Comment:
Nitpick: I think `usingConfigFile` or just `configFile` would be better naming for a feature flag.